### ISSUE TITLE
Trouble with pagination

### ISSUE THREAD

**Issue by tungol** [schubergphilis/oktalib#26]
While querying the `/api/v1/apps` endpoint, Okta is returning continuation links with limit parameters already applied, e.g. `https://okta.com/api/v1/apps?after=0oa1g2o8sr2hhAGBh0h8&limit=100`

That's generating an error for me when oktalib tries to GET it while also supplying the limit as an additional parameter. It works fine as long as I make sure that the limit parameter is only provided once.

Here's the error I get:

{'errorCode': 'E0000037', 'errorSummary': "Type mismatch exception.  Could not convert parameter 'limit' to the required type.", 'errorLink': 'E0000037', 'errorId': 'oaeAbUMna8vR_qMMx4xZ5iGyw', 'errorCauses': []}

I'm using a quick and dirty fix in Okta._get_paginated_url that looks like this:

```
            while next_link:
                if "&limit=" in next_link:
                    response = self.session.get(url=next_link)
                else:
                    response = self.session.get(url=next_link, params=params)
                results.extend(response.json())
```

Which seems to work as expected, but could be more correct if it actually parsed out the query string.

**Comment by costastf** [comment]
Can you please give https://github.com/schubergphilis/oktalib/tree/fix-limit a try and let me know if this fixes it for you? @tungol

**Comment by costastf** [comment]
This should have been fixed with v2.0.0 so closing for now.
